cobra/flux_analysis/reaction.py

```diff
@@ -36,7 +41,8 @@ def assess(model, reaction, flux_coefficient_cutoff=0.001):
         return results
 
 
-def assess_precursors(model, reaction, flux_coefficient_cutoff=0.001):
+def assess_precursors(model, reaction, flux_coefficient_cutoff=0.001, 
```

Comment:
There is a trailing white space after the comma PEP8 is complaining about, sorry :(